lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,6 +55,17 @@
   --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
   --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
 
+  // NxGlobalHeader
+  --nx-color-global-header-border: var(--nx-color-border);
+
+  //NxPageHeader variables
+  --nx-color-page-header-background: var(--nx-color-component-background);
+  --nx-color-page-header-border-bottom: var(--nx-color-border);
+  --nx-color-page-header-divider: var(--nx-swatch-grey-30);
+  --nx-color-page-header-product-name: var(--nx-color-text);
```

Comment:
Not 100% sure about this, but what are your thoughts about adding `-text` at the end so it is consistent with the other text colors?

`--nx-color-page-header-product-name-text`

Reply:
> Not 100% sure about this, but what are your thoughts about adding `-text` at the end so it is consistent with the other text color variables?
> 
> `--nx-color-page-header-product-name-text`

Makes sense! Updated here: [3d58df9](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3d58df9ed3a614bbb6dbd7d27a441f00c3fda659)